pom.xml

```diff
@@ -33,17 +33,50 @@
   </ciManagement>
 
   <properties>
+    <java.version>21</java.version>
+    <maven.compiler.source>21</maven.compiler.source>
+    <maven.compiler.target>21</maven.compiler.target>
+
     <module_name>mod-patron</module_name>
     <http.port>8081</http.port>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <ramlfiles_util_path>${ramlfiles_path}/raml-util</ramlfiles_util_path>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <aspectj.version>1.9.19</aspectj.version>
-    <raml-module-builder.version>35.3.0</raml-module-builder.version> <!-- also update vertx.version -->
-    <vertx.version>4.5.10</vertx.version>
+    <aspectj.version>1.9.22.1</aspectj.version>
+    <raml-module-builder.version>35.4.0</raml-module-builder.version> <!-- also update vertx.version -->
+    <vertx.version>4.5.13</vertx.version>
+
+    <!--Dependencies properties-->
+    <lombok.version>1.18.36</lombok.version>
+    <commons-lang3.version>3.17.0</commons-lang3.version>
+    <slf4j.version>1.7.30</slf4j.version>
+    <joda-time.version>2.10.9</joda-time.version>
+    <rest-assured.version>5.5.1</rest-assured.version>
+    <junit.version>4.13.2</junit.version>
+    <hamcrest.version>2.2</hamcrest.version>
+    <wiremock.version>2.25.1</wiremock.version>
+    <system-stubs-jupiter.version>2.1.6</system-stubs-jupiter.version>
+
+    <!--Dependencies management properties-->
+    <junit-bom.version>5.11.2</junit-bom.version>
+    <log4j-bom.version>2.24.3</log4j-bom.version>
+
+    <!--Plugin properties-->
+    <build-helper-maven-plugin.version>3.6.0</build-helper-maven-plugin.version>
+    <jacco-maven-plugin.version>0.8.12</jacco-maven-plugin.version>
+    <aspectj-maven-plugin.version>1.14</aspectj-maven-plugin.version>
+    <properties-maven-plugin.version>1.0.0</properties-maven-plugin.version>
```

Comment:
can be 1.2.1
https://github.com/folio-org/raml-module-builder/blob/v35.4.0/pom.xml#L337